File: frontend/lib/models/education.dart
class Education {
  final String degree;
  final String institution;
  final String location;
  final DateTime startDate;
  final DateTime endDate;
  final String photoUrl;

  Education({
    required this.degree,
    required this.institution,
    required this.location,
    required this.startDate,
    required this.endDate,
    required this.photoUrl,
  });

  factory Education.fromJson(Map<String, dynamic> json) {
    return Education(
      degree: json['degree'],
      institution: json['institution'],
      location: json['location'],
      startDate: DateTime.parse(json['startDate']),
      endDate: DateTime.parse(json['endDate']),
      photoUrl: json['photoUrl'],
    );
  }
}

File: backend/src/main/java/github/backend/model/Education.java
package github.backend.model;

import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDate;

@Data
@Entity
@Table(name = "edcuation")
public class Education {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int id;

    private String degree;
    private String institution;
    private String location;

    @Column(name = "start_date")
    private LocalDate startDate;

    @Column(name = "end_date")
    private LocalDate endDate;

    @Column(name = "photo_url")
    private String photoUrl;

}
